src/content/docs/journey-large-logs/get-started.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: How to manage large log ingest 
+metaDescription: "Test"
+---
+
+import logsParsing from 'images/logs_screenshot_full-parsing.webp'
+import logsPartition from 'images/logs_screenshot_full-partition.webp'
+
+
+Modern systems create large amounts of log data. You might be dealing with hundreds of gigabytes to dozens of terabytes today, and the amount will continue to increase as your system scales. Anything past a few gigabytes of logs means hours of toil while you search through logs when need arises. Log management solutions, like New Relic, provide the tools to handle large sets of logs and make them managable — and more important, valuable. 
+
+This tutorial walks you through how to us New Relic to manage a large amount of log ingest. You'll start by forwarding your logs to New Relic, which means sending your log data to New Relic automatically. You'll then identify what logs to ingest and which to drop. Finally you'll organize your logs through partitions and parsing.
+
+
+<img
+    title="log-dashboard"
+    alt="An image displaying New Relic's log monitoring dashboard"
+    src={journeyLogsDash}
+/>
+
+## Choosing a log management platform [#choose]
+
+
+Once you've identified you have a problem with managing logs, it's time to choose a log management platform. There are many platforms out there. Some focus on quick automation but sacrifice ease-of-use. Others focus on complex features, but obscure their pricing.
+
+New Relic's philosphy when it comes to log management focuses on three things: we want out logs solution to be **flexible, transparent, and usage based**. Let's quickly talk about what these mean:
```

Comment:
If we say "three things" we should probably have three bullets below and break out `transparent` and `usage-based`